source/features/next-scheduled-github-action.tsx

```diff
@@ -10,7 +10,7 @@ import * as api from '../github-helpers/api';
 import {getRepo} from '../github-helpers';
 
 const getScheduledWorkflows = cache.function(async (): Promise<Record<string, string> | false> => {
-	const {repository: {object: {entries: workflows}}} = await api.v4(`
+	const {repository: {object}} = await api.v4(`
 		repository() {
 			object(expression: "HEAD:.github/workflows") {
```

Comment:
So we don't have to name it `object`
```suggestion
			workflows: object(expression: "HEAD:.github/workflows") {
```